airbyte-integrations/connectors/source-webflow/acceptance-test-config.yml

```diff
@@ -0,0 +1,20 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-webflow:0.1.0-alpha
+tests:
+  spec:
+    - spec_path: "source_webflow/spec.yaml"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+# Remove the following, as it causes integration tests to fail
+#    - config_path: "integration_tests/invalid_config.json"
+#      status: "failed"
```

Comment:
@alexander-marquardt can you remove these comments after adding a proper invalid_config.json?

Reply:
Sure - what should a proper invalid_config.json look like?

Reply:
See my comment above and let me know if it makes sense!

Reply:
Done